devices/baseEstimatorV1/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+# Copyright (C) 2019 Istituto Italiano di Tecnologia (IIT). All rights reserved.
+# This software may be modified and distributed under the terms of the
+# GNU Lesser General Public License v2.1 or any later version.
+
+set(iDynTree_REQUIRED_VERSION 0.11.0)
+
+set(CMAKE_INCLUDE_CURRENT_DIR TRUE)
```

Comment:
Is this necessary?

Reply:
Removed in 82d978f1892d49beb5bdc29df0f343a153b2678a